### PR TITLE
Move graphing to core

### PR DESCRIPTION
**Pull request by wpride**
cross-request: https://github.com/dimagi/commcare-core/pull/575

### CODE REVIEW

**Comment by wpride** [comment]
@damagatchi retest this please

**Comment by amstone326** [comment]
@damagatchi retest this please

**Comment by amstone326** [comment]
@wpride Build failure is because of a test failure

**Comment by wpride** [comment]
@damagatchi retest this please

**Comment by wpride** [comment]
@damagatchi retest this please

File: app/unit-tests/src/org/commcare/android/tests/processing/FormStorageTest.java [hunk 1]
```diff
@@ -120,7 +120,7 @@
             , "org.commcare.suite.model.graph.BubbleSeries"
             , "org.commcare.suite.model.graph.Graph"
             , "org.commcare.suite.model.graph.XYSeries"
-            , "org.commcare.xml.DummyGraphParser$DummyGraphDetailTemplate"
+            //, "org.commcare.xml.DummyGraphParser$DummyGraphDetailTemplate"
```
Comment (amstone326) [inline app/unit-tests/src/org/commcare/android/tests/processing/FormStorageTest.java]: @wpride Why did this get commented out?

Comment (wpride) [inline app/unit-tests/src/org/commcare/android/tests/processing/FormStorageTest.java]: This class was removed - will add a comment

**Comment by wpride** [comment]
@damagatchi retest this please

**Pull request merged**